plugins/nordic-hid/README.md

```diff
@@ -83,9 +85,11 @@ This plugin requires ioctl `HIDIOCSFEATURE` and `HIDIOCGFEATURE` access.
 
 This plugin has been available since fwupd version `1.7.3`.
 
+Since the [nRF Connect SDK v2.7.0 release](https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/releases_and_maturity/releases/release-notes-2.7.0.html), the format version of the `dfu_application.zip` file generated by the build system was updated from `0` to `1`.
+The format version `1` is supported by the fwupd since the `1.9.25` release.
```

Comment:
```suggestion
The format version `1` is supported by the fwupd since the 1.9.25 release.
```

Reply:
I prefer to keep the syntax here (that's how fwupd version is referred to in other places)